src/applications/disability-benefits/all-claims/pages/individualsInvolvedFollowUp.js

```diff
@@ -90,16 +90,12 @@ export const schema = index => ({
                 properties: {
                   first: {
                     type: 'string',
-                    minLength: 1,
-                    maxLength: 30,
```

Comment:
Will this cause any problem down the line if a user enters a name that's 3,000 characters long?

Reply:
Due to the sensitivity of the form, the backend will generate an overflow page for anything that exceeds the limit for the form field (PDF form field).